db/migrate/20180309144814_rename_number_of_issues_to_feedex_comments.rb

```diff
@@ -0,0 +1,6 @@
+class RenameNumberOfIssuesToFeedexComments < ActiveRecord::Migration[5.1]
+  def change
+    rename_column :facts_metrics, :feedex_comments, :feedex_comments
```

Comment:
Not sure the rename is valid?

Reply:
Sorry @pmanrubia, well spotted.  I should not have used `find_and_replace`! 😂 😔